### PR TITLE
small fix to repeated text and fix to github issue

### PR DESCRIPTION
**Pull request by ayushr2**
Fix to illinois-cs241/assignments#590 and a small bug with duplicated text

### CODE REVIEW

File: password_cracker.md [hunk 1]
```diff
@@ -192,6 +192,13 @@ This is an exercise we have intentionally left for the reader, but one hint we w
 **Remember to use appropriate synchronization, and make sure to use `crypt_r`.**
 If you create a new thread for each task (instead of keeping the threads in the thread pool running), you will lose points! (and your implementation will be very slow)
 
+### Bounds
+1 <= thread_pool_size <= 8
+
+1 <= number of passowrds <= 12
```
Comment (aneeshdurg) [inline password_cracker.md]: *passwords

File: password_cracker.md [hunk 2]
```diff
@@ -302,6 +309,13 @@ When the worker threads finish a task, each thread will print the number of pass
 After all worker threads finish each task, the main thread will print the password (if found), the total number of hashes, the wall clock and CPU time spent on that task, and the ratio of CPU time to wall clock time.
 Note that we have not provided any of the timing print statements in `cracker2`.
 
+### Bounds
+1 <= thread_pool_size <= 13
```
Comment (aneeshdurg) [inline password_cracker.md]: Is this here twice?

Comment (ayushr2) [reply password_cracker.md]: Nope, added bounds for both version 1 and 2. Tests were significantly different.

Comment (ayushr2) [reply password_cracker.md]: Combined into one now

**Pull request merged**